js/collections/Results.js

```diff
@@ -30,6 +30,11 @@ export default class extends Collection {
       updatedResult.type = result.type;
       const relationships = result.relationships || {};
 
+      if (!window.zoom) {
```

Comment:
Another example of me mucking an unrecognized currency state. This will make the first listing card in a set of search results have an unrecognized currency, resulting in:

![image](https://user-images.githubusercontent.com/794517/32811913-1ea9ffa6-c957-11e7-899a-69258e8e7063.png)